app/component/ItineraryPage.js

```diff
@@ -472,12 +465,10 @@ class ItineraryPage extends React.Component {
       date: earliestArrivalTime.format('YYYY-MM-DD'),
       time: earliestArrivalTime.format('HH:mm'),
     };
-    this.setState({
-      loadingMore: reversed ? spinnerPosition.bottom : spinnerPosition.top,
-    });
-    this.showScreenreaderLoadingAlert();
+    this.setState({ loadingMore: reversed ? 'bottom' : 'top' });
```

Comment:
Why do we go back to hardcoding strings here?

Reply:
Same as above. A bit worrying. Restored.